src/lib/y2storage/partition_tables/base.rb

```diff
@@ -116,6 +116,16 @@ class Base < Device
       #   @return [Boolean] whether a partition can have this partition id.
       storage_forward :partition_id_supported?
 
+      # @!method align(region, align_policy = AlignPolicy::ALIGN_END, align_type = AlignType::OPTIMAL)
+      #   Align the region according to align policy and align type.
```

Comment:
Does this return an new region that is aligned without modifying the passed one (ruby style) or does it modifies the passed argument (libstorage-ng style)? I would like to know from the documentation.

Reply:
Yes, it always returns a new object. Commented.